__tests__/app.test.js

```diff
@@ -197,3 +197,38 @@ describe("GET - /api/reviews/:review_id (comment count)", () => {
       });
   });
 });
+
+describe("GET - /api/reviews", () => {
+  test("Status 200- should respond with an array of review objects", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body: { reviews } }) => {
+        expect(reviews).toBeInstanceOf(Array);
+        expect(reviews).toHaveLength(13);
+        expect(reviews).toBeInstanceOf(Array);
+        reviews.forEach((review) => {
+          expect.objectContaining({
+            review_id: expect.any(Number),
+            title: expect.any(String),
+            review_body: expect.any(String),
```

Comment:
Please read this ticket very carefully and check that what you have matches what we are asking for.